src/main/java/lessons/hw2/IsParenthesisValid.java

```diff
@@ -0,0 +1,61 @@
+package lessons.hw2;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+
+public class IsParenthesisValid {
+
+    public static boolean isValid(String str) {
+
+        if (str.isEmpty()) {
+            throw new RuntimeException("String is empty");
+        }
+
+        if (str.length() > 10000) {
+            throw new RuntimeException("String is out of range");
+        }
+
+        Map<Character, Character> pairs = new HashMap<Character, Character>();
+        pairs.put('(', ')');
+        pairs.put('{', '}');
+        pairs.put('[', ']');
+
+        Stack<Character> deck = new Stack<>();
+        char[] chArr = str.toCharArray();
```

Comment:
Since Java is strongly typed language, we can omit mentioning data type in a variable name. 
```suggestion
        char[] chars = str.toCharArray();
```

Reply:
+